sections/errorhandling/centralizedhandling.md

```diff
@@ -2,7 +2,7 @@
 
 ### One Paragraph Explainer
 
-Without one dedicated object for error handling, greater are the chances of important errors hiding under the radar due to improper handling. The error handler object is responsible for making the error visible, for example by writing to a well-formatted logger, sending events to some monitoring product like [Sentry](https://sentry.io/), [Rollbar](https://rollbar.com/), or [Raygun](https://raygun.com/). Most web frameworks, like [Express](http://expressjs.com/en/guide/error-handling.html#writing-error-handlers), provide an error handling middleware mechanism. A typical error handling flow might be: Some module throws an error -> API router catches the error -> it propagates the error to the middleware (e.g. Express, KOA) who is responsible for catching errors -> a centralized error handler is called -> the middleware is being told whether this error is an untrusted error (not operational) so it can restart the app gracefully. Note that it’s a common, yet wrong, practice to handle errors within Express middleware – doing so will not cover errors that are thrown in non-web interfaces.
+Without one dedicated object for error handling, greater are the chances for incosistent errors handling: Errors that are thrown within web requests might get handled diffrently from those are raised during the startup phase and those which are raised by scheduled jobs. This might lead to some type of errors being handled impoperly. This single error handler object is responsible for making the error visible, for example by writing to a well-formatted logger, firing metrics using some monitoring product (like [Prometheus](https://prometheus.io/), [CloudWatch](https://aws.amazon.com/cloudwatch/), [DataDog](https://www.datadoghq.com/), and [Sentry](https://sentry.io/)) and to decide whether the process should crash. Most web frameworks provide an error catching middleware mechanism - A typical mistake is to place the error handling code within this middleware. By doing so, you won't be able to reuse the same handler for errors that are caught in different scenarios like scheudled jobs, message queue subscriber, and uncaught exceptions. Consequently, the error middleware should be used only to catch erros and forward to the handler. A typical error handling flow might be: Some module throws an error -> API router catches the error -> it propagates the error to the middleware (e.g. or to other mechanism for catching request-level error) who is responsible for catching errors -> a centralized error handler is called.
```

Comment:
Small typo: incosistent => inconsistent. 
Also, I'm not sure we need the `s` in `errors handling`

Reply:
from those ~are~ raised during

Reply:
impoperly => improperly

Reply:
`scheudled jobs` => `scheduled`

Reply:
`should be used only to catch erros` => `errors`

Reply:
@kevynb Great catches, apparently my grammar checker tool (Ginger) was not active. Thanks!